core/src/main/scala/org/ensime/core/Project.scala

```diff
@@ -53,19 +52,9 @@ class Project(
   }
   private val classfileWatcher = context.actorOf(Props(new ClassfileWatcher(config, searchService :: reTypecheck :: Nil)), "classFileWatcher")
 
-  def receive: Receive = awaitingConnectionInfoReq
+  def receive: Receive = handleRequests
 
-  def awaitingConnectionInfoReq: Receive = withLabel("awaitingConnectionInfoReq") {
-    case ConnectionInfoReq =>
-      sender() ! ConnectionInfo()
```

Comment:
it looks like we're not sending `ConnectionInfo` at all now. Should you maybe send it in the server connection-handler actor?